portfolio.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="homestyle.css">
+  </head>
+  <body>
+	  <header class="navbar">
+		  <div id="NC"><a href="home.html">Nick Clifford</a> 
+		  </div>
+		  <div class="links">
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>
+		  
+	  </header>
+	  <div class="portmain">
+		  <div class="porttitle">Things I have made...
+		  </div>
+		  <div class=portcontents>
+			  <div class="port-container"></div>
+		  </div>
+
+	  </div>
+	<div class="footer">
```

Comment:
Do you want your footer to come at the bottom of the page? As currently it comes in as soon as all preceding content is finished.